drivers/SRCommand.py

```diff
@@ -83,7 +83,7 @@ def parse(self):
         #                                  priority=util.LOG_DEBUG )
 
         try:
-            params, methodname = xmlrpc.client.loads(sys.argv[1])
+            params, methodname = xmlrpc.client.loads(os.fsencode(sys.argv[1]))
```

Comment:
This is a string passed on the commandline, is `fsencode` the correct thing to be using as it's not really just a filename but actually a stream of XML data. I understand that we need to do utf-8 encoding/decoding but maybe not with `fsencode` in case it has any special behaviour?

Reply:
The Python documentation for sys.argv says:

> On Unix, command line arguments are passed by bytes from OS. Python decodes them with filesystem encoding and “surrogateescape” error handler. When you need original bytes, you can get it by `[os.fsencode(arg) for arg in sys.argv]`.

I think `fsencode` is a bit of a confusing name in this context.